libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts

```diff
@@ -141,9 +141,7 @@ export function generateCohortsStatsTable(
             // only 1 unique value in the set, set color to 0
             colorValue = 0;
           }
-          const colorConfig = { color: "transparent" };

```

Comment:
Could we conditionally keep this for MO metrics (in dataset cohorts)?

Reply:
I've turned off the heatmap for OD. See updates :-)